script.js

```diff
@@ -596,11 +675,45 @@ function handleRentStrikeLayer(geoJson) {
   return rentStrikeLayerMarkers;
 }
 
-// Ensures that map overlay pane layers are displayed in the correct Z-Order
-function fixZOrder(dataLayers) {
-  dataLayers.forEach(function (layerGroup) {
-    if (map.hasLayer(layerGroup)) {
-      layerGroup.bringToBack();
-    }
+// Do not add nations to map at start
+function handleNationsLayer(geojson) {
+  // Scores are bound to range prop of each feature
+  const layerOptions = {
+    style: feature => {
+      const score = feature.properties.range
+      return {
+        // Get color from score
+        fillColor: scoreFillColors[score] || '#808080', // fallback color
```

Comment:
@river-honer maybe just check that #808080 isn't too dark where it might cover the basemap too much.

Reply:
Done, picked lighter color, based on personal preference.